app/javascript/controllers/media_player_controller.js

```diff
@@ -20,6 +20,12 @@ export default class extends Controller {
         const event = new CustomEvent('time-update', { detail: timestamp })
         window.dispatchEvent(event)
       })
+
+      // patch for https://github.com/sul-dlss/sul-embed/issues/2258
+      // For some reason loadedmetadata is being triggered before texttrackchange.
+      // https://github.com/videojs/video.js/blob/main/src/js/tracks/text-track-display.js#L127-L128
+      // This is not true in video.js. This patch will fix the problem.
```

Comment:
```suggestion
      // This behavior does not occur in plain video.js. Calling updateDisplay() explicitly fixes the problem in this app.
```

Reply:
I am happy to update this comment. Do you have an answer to the question above. It looks like you are the one who implemented that listener and the comment isn't giving me any clarity because I am not quite sure how timeupdate could be available before loadedmetadata is done firing.

Reply:
Wasn't it @mjgiarlo https://github.com/sul-dlss/sul-embed/commit/8c4dba7a0cbee3dcd25946c473cdd98af31999ba ?

Reply:
right. i read the code wrong. thanks

Reply:
@jcoyne I talked to Mike. It seems that the listener was for autoscroll and moving it out of timeupdate doesn't affect autoscroll and fixes the bug outlined in the issue. This PR is ready for review.